website/docs/faqs/Project/delete-a-project.md

```diff
@@ -0,0 +1,17 @@
+---
+title: How do I delete a project?
```

Comment:
do we have steps for this for core users? if so, i think it would make sense to add it here too and have a core and cloud section. however if the audience is specifically for cloud users, i suggest clarifying it in the title so the user is aware from the get go.

```suggestion
title: How do I delete a project in dbt Cloud?
```